### ISSUE TITLE
[$30][MSFT-99] Challenges page: Screen reader doesn't announce the type of element for the 'filter' control

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2804]
**Steps to Reproduce**
1. Go to the Challenges page: https://www.topcoder.com/challenges. 2. Ensure the screen reader is switched on. 3. Press the down arrow from the 'search challenges' text box a couple of times. 4. Notice the screen reader just reads 'filter' instead of 'filter button'.

**Expected Result**
Screen reader should announce the type of element in this case as 'filter button'.

**Actual Result**
Screen reader announces the 'filter' control as 'filter' instead of 'filter button'. The role of the element is not announced.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 4. Robust
**WCAG Level**: A
**WCAG Success Criterion**: 4.1.2 Name, Role, Value
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%206

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096061 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096061 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by r0hit-gupta** [comment]
Please assign me

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096061 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by asadath1395** [comment]
@veshu This is already fixed i guess

**Comment by veshu** [comment]
Yes,  label is not updated

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097668 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097668 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by codeMinter** [comment]
@veshu this issue is working fine on base repo code. Also PR raised by @r0hit-gupta has no changes hence closed PR. Please take it ahead for QA verification. 

@sushilshinde FYI ☝️

**Comment by r0hit-gupta** [comment]
@codeMinter some other PR was merged with the same changes. Thus it showed 0 changes in my PR after that.

**Comment by drasticdpk** [comment]
Verified on Windows/Chrome/JAWS

**Comment by lakshmiathreya** [comment]
Verified on Win10/Chrome/NVDA and MacOS/Chrome Voice Over. Filter button ready out as expected. Closing Issue.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097668<br/><br/>```This is an automated message for crazyk via Topcoder X```
